app.js

```diff
@@ -13,22 +13,31 @@ var config = {
 
 require('./index.js').connect(function (context) {
 
+    context.fs.readdir(context.path.join(__dirname, 'api', 'models'), (err, files) => {
+        //Load ALL modules.
+        //Don't get confused by weird logging order! console.log is async and will mix up the actual loading order.
+        files.forEach(file => {
```

Comment:
Sorry if I'm mistaking something but in this case it is not ensured that the forEach loop will be finished loading all the models before we load the buildRelations module, right? http://stackoverflow.com/questions/18983138/callback-after-all-asynchronous-foreach-callbacks-are-completed
But anyway, I don't think we even need. I think that sequelize.sync takes care of [enforcing the correct order](https://github.com/sequelize/sequelize/blob/3e5b8772ef75169685fc96024366bca9958fee63/lib/sequelize.js#L972) of the table and relation creations, independent of how they are loaded beforehand.